### PR TITLE
Update README per new API

### PR DESCRIPTION
**Pull request by pselle**
@XerTheSquirrel I'm unsure of what the API should look like for the "using service directly" example

### CODE REVIEW

**Comment by XerTheSquirrel** [comment]
The "using it directly" section changes a few details:

Instead of `Supplier<R>` it is `Function<IOpipeExecution, R>`.

And the lambda changes from `() -> foo` to `(exec) -> foo`.

As seen at <https://github.com/iopipe/iopipe-java-core/blob/master/example/src/main/java/com/iopipe/examples/ManualHello.java>.

**Comment by pselle** [comment]
Updated, thanks @XerTheSquirrel !

**Comment by XerTheSquirrel** [comment]
Also, the parameters to `run()` is the `Context` instance (so it is `run(Context, Function<IOpipeExecution, R>)`.

So the call should be `service.<String>run(context, (exec) -> "Hello World!");`.

File: README.md [hunk 1]
```diff
@@ -91,27 +91,29 @@ There are three ways to use the service:
 This class provides an implementation of `RequestHandler<I, O>`.
 
  * Add the following import statement:
+   * `import com.iopipe.IOpipeExecution;`
    * `import com.iopipe.SimpleRequestHandlerWrapper;`
  * Add a class which extends:
    * `SimpleRequestHandlerWrapper<I, O>`
  * Implement the following method:
-   * `protected O wrappedHandleRequest(I __input, Context __context)`
+   * `protected O wrappedHandleRequest(IOpipeExecution __exec, String __input)`
```
Comment (XerTheSquirrel) [inline README.md]: `String` should be `I` because it is a generic type.

**Pull request merged**